test/test_utils.py

```diff
@@ -12,7 +12,7 @@ class TestSetupReadline(unittest.TestCase):
     class NameSpace(object):
         pass
 
-    def __init__(self, *args, **kwargs):
+    def setUp(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
```

Comment:
Should the call to `super()` not also be changed to `super().setUp(*args, **kwargs)`?

Reply:
Oh thanks Peter! Yes that would be a very welcome change. I suspect this is just an oversight...